projects/cdk/schematics/ng-update/index.ts

```diff
@@ -93,6 +94,23 @@ function addTaigaStyles(options: Schema): Rule {
             version: TAIGA_VERSION,
```

Comment:
oh please remove "addPackageJsonDependency", it's my mistake, I moved it into "addStylesToAngularJson" in previous hotfix-pr but forget to remove here

Reply:
done ✅